apps/meeting/package.json

```diff
@@ -2,21 +2,24 @@
   "name": "chime-sdk-meeting-demo",
   "version": "0.1.0",
   "dependencies": {
+    "@aws-sdk/client-chime-sdk-meetings": "^3.485.0",
     "@types/react": "^18.2.7",
     "@types/react-dom": "^18.2.4",
     "@types/styled-components": "^4.4.3",
     "@types/webpack-env": "^1.17.0",
     "amazon-chime-sdk-component-library-react": "file:../../amazon-chime-sdk-component-library-react",
     "amazon-chime-sdk-js": "^3.18.2",
-    "aws-sdk": "^2.1354.0",
+    "body-parser": "^1.20.2",
     "fs-extra": "^10.1.0",
     "lodash.isequal": "^4.5.0",
+    "morgan-body": "^2.6.9",
```

Comment:
These two is a express middleware to log the http request and response automatically. For example:
```
Request: POST /end at Wed Jan 17 2024 10:27:37 GMT-0800, IP: ::ffff:127.0.0.1, User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/120.0.0.0 Safari/537.36
Request Body:
{
    "title": "d1"
}
Response: 200 150.609 ms 
```

Reply:
Nice. I assume this logs to terminal?
Do you want to do this for JS SDK demo too :))

Reply:
Yeah, it logs these to terminal that runs the server. I wish I could have time to do this for JS SDK demo. That one is more complicated than this one.